### DISCUSSION TITLE
Draw.io diagram of the enterprise scale architecture for AKS

### DISCUSSION THREAD

**Discussion by Andysteiner** [Azure/Enterprise-Scale#1595] [Show and tell]
I needed to have an adjustable draw.io drawing of the enterprise scale architecture for AKS as shown here: https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/app-platform/aks/landing-zone-accelerator#example-conceptual-reference-architecture 
As I could not find any, I draw it myself. I could imagine that others might be interested in that as well, so I would like share it. Where is the best place? This repo?

**Comment by jtracey93** [comment]
Hey please raise this as an issue over on the AKS Accelerator Repo https://github.com/Azure/AKS-Landing-Zone-Accelerator
